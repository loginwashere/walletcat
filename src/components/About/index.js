import React from 'react'
import { intlShape, injectIntl, defineMessages } from 'react-intl'

const messages = defineMessages({
  header: {
    id: 'About.header',
    defaultMessage: 'About',
    description: 'About header'
  }
})

const About = ({ intl }) => (
  <h1>{intl.formatMessage(messages.header)}</h1>
)

About.propTypes = {
  intl: intlShape.isRequired,
}

export default injectIntl(About)
