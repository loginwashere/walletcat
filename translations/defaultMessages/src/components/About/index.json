[
  {
    "id": "About.header",
    "description": "About header",
    "defaultMessage": "About"
  }
]