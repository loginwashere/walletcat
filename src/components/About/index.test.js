import React from 'react'
import renderer from 'react-test-renderer'
import About from '.'

describe('components:About:', () => {
  it('renders without crashing', () => {
    const tree = renderer.create(
      <About />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
