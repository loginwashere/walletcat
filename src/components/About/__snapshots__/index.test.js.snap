exports[`components:About: renders without crashing 1`] = `
<h1>
  About
</h1>
`;
